signatures/golang/stdio_over_socket.go

```diff
@@ -51,6 +51,12 @@ func (sig *stdioOverSocket) GetSelectedEvents() ([]detect.SignatureEventSelector
 	}, nil
 }
 
+func (sig *stdioOverSocket) GetFilters() ([]protocol.Filter, error) {
+	return []protocol.Filter{
+		{Field: "event", Operator: protocol.Equal, Value: []interface{}{"security_socket_connect", "dup", "dup2", "dup3", "close", "sched_process_exec"}},
```

Comment:
we could possibly add a `equals` or `compare` function to types like addresses and then be able to compare them.

Reply:
Not sure I understand, could you please elaborate and provide example syntax?

Reply:
If we have for example the slimCreds struct we could define a function named compare that receives another variable and compares them using some logic
`slimCred1.Compare(bitmaskIntFromFilter)`

So we could compare more complex types and not only strings and ints.

Reply:
So SlimCred for example are usually translated to args right? You could set filters on the corresponding args created from it I think. So you could create your own helpers to make filters for those structs.
It would really help if I got a code example, or you can make a PR to this branch to show an example.

Reply:
I just added an example of relevant filters here @AsafEitani could you check?